addons/stock/views/stock_package_level_views.xml

```diff
@@ -42,6 +42,10 @@
                         </tree>
                     </field>
                 </group>
+                <footer class="oe_edit_only">
+                    <button string="Confirm" special="save" class="oe_highlight"/>
+                    <button string="Discard" special="cancel"/>
+                </footer>
```

Comment:
Also appear in the usual form view, where these buttons have no effect (doesn't close the window) and hase already a close button.